docs/config_reference.rst

```diff
@@ -456,6 +456,20 @@ System Partition Configuration
       No other test would be able to proceed.
 
 
+.. py:attribute:: systems.partitions.sched_options.sacct_retries
```

Comment:
Maybe we should not name it "retries" as this somehow implies that once the command fails we continue retrying it. What if we call it

```suggestion
.. py:attribute:: systems.partitions.sched_options.max_sacct_failures
```